File: src/main/webapp/js/Fakemail/model/Email.js
Ext.define('Fakemail.model.Email', {
	extend: 'Ext.data.Model',
	fields: [
		{ name: 'id', type: 'int' },
		{ name: 'sender', type: 'string' },
		{ name: 'recipient', type: 'string', useNull: true },
		{ name: 'dateSent', type: 'date' },
		{ name: 'subject', type: 'string' }
	]
});

File: src/jpaModelgen/java/org/legurun/test/fakemailserver/model/Email_.java
package org.legurun.test.fakemailserver.model;

import java.util.Date;
import javax.annotation.Generated;
import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;

@Generated(value = "org.hibernate.jpamodelgen.JPAMetaModelEntityProcessor")
@StaticMetamodel(Email.class)
public abstract class Email_ extends org.legurun.test.fakemailserver.model.AbstractEntity_ {

	public static volatile SingularAttribute<Email, Date> sentDate;
	public static volatile SingularAttribute<Email, Sender> sender;
	public static volatile SingularAttribute<Email, String> subject;
	public static volatile SingularAttribute<Email, String> recipient;
	public static volatile SingularAttribute<Email, byte[]> message;

}

